dev/aliases

```diff
@@ -83,7 +83,7 @@ alias comDR='comRD && comRD && comPD'
 alias comXR='comR  && comR  && comP'
 alias comXS='comS  && comP'
 alias   ent='comR  && entR  && entS  && entP  && entT'
-alias  entD='comR  && entRD && entSD && entPD'
+alias  entD='comRD && entRD && entSD && entPD'
 alias  entX='comR  && entR  && entS  && entP'
```

Comment:
It appears that the alfresco-enterprise-share build has problems building AGS if the TAS base libraries are not already built. Included in the 'docker' build of community.